lua/entities/gmod_wire_expression2/core/selfaware.lua

```diff
@@ -206,26 +206,29 @@ end
 --[[******************************************************************************]]--
 -- Extensions
 
+local getExtensionStatus = E2Lib.GetExtensionStatus
+local e2Extensions
+local e2ExtensionsTable
+-- See postinit for these getting initialized
+
 [nodiscard]
 e2function array getExtensions()
```

Comment:
Same as below, should be ~30 ops, or dynamic based on how many extensions there are